ionics_fits/models/sinusoid.py

```diff
@@ -279,11 +279,11 @@ def estimate_parameters(self, x: TX, y: TY):
         # bounds across
         if (
             self.parameters["y0"].has_user_initial_value()
-            and self.parameters["a"].has_user_initial_value()
+            and sine.parameters["a"].has_user_initial_value()
```

Comment:
Note to self, this change does not have any functional impact since we have already transferred all metadata associated with `a` from `self` to `sine`